exasol/python_extension_common/deployment/language_container_validator.py

```diff
@@ -27,25 +27,32 @@ def _create_dummy_udf(conn: pyexasol.ExaConnection, language_alias: str,
 
     udf_name = _get_test_udf_name(schema)
     sql = dedent(f"""
-    CREATE OR REPLACE {language_alias} SCALAR SCRIPT {udf_name}()
-    RETURNS DECIMAL(1, 0) AS
+    CREATE OR REPLACE {language_alias} SET SCRIPT {udf_name}(i DECIMAL(10, 0))
+    RETURNS DECIMAL(10, 0) AS
 
     def run(ctx):
-        return 0
+        return ctx.i
```

Comment:
maybe to make sure we actually run on all nodes, it is useful if we return the node id of the UDF `exa.meta.node_id`